README.md

```diff
@@ -1,6 +1,21 @@
 # Board&Roll
 
-Board&Roll is an application for finding and organizing board game meetings in your local area. Main goal is to help people who love board games.
+## Description
+**Board&Roll** is a web application for organizing and finding board game meetings in your local area. User can choose a game, create an event and invite friends or he can join one of the events nearby.
+
+
+## Installation
+To download code from Github use command </br>
+`git clone https://github.com/Ragnarson-Internship2016/group-III.git ` </br>
```

Comment:
Imho it should not contain instruction how to clone repository. Just a sentence like "Clone repository from `...`"